source/languages/en/dataplatform/using-dataplatform/configuration/setup-a-cluster.md

```diff
@@ -239,37 +242,36 @@ The hostnames you enter must match what you set as `SPARK_MASTER_IP` in priv/spa
 
 ####Redis
 
-To add a Redis service to a node:
+To register the service configuration for Redis, issue the following command:
 
-```shell
-data-platform-admin add-service-config my-redis redis HOST="0.0.0.0" REDIS_PORT="6379"
+```bash
+sudo data-platform-admin add-service-config my-redis redis HOST="0.0.0.0" REDIS_PORT="»REDIS_PORT«"
 ```
 
 ####Cache Proxy
 
-To add a cache proxy service to a node:
+To register the service configuration for Cache Proxy, issue the following command:
 
-```shell
-data-platform-admin add-service-config my-cache-proxy cache-proxy  HOST="0.0.0.0" CACHE_PROXY_PORT="11211" CACHE_PROXY_STATS_PORT="22123" CACHE_TTL="15s" RIAK_KV_SERVERS="»IPADDRESS«:»PROTOCOLBUFFERSPORT« »IPADDRESS2«:»PROTOCOLBUFFERSPORT« »IPADDRESS3«:»PROTOCOLBUFFERSPORT«" REDIS_SERVERS="»IPADDRESS«:6379 »IPADDRESS2«:6379 »IPADDRESS3«:6379"
+```bash
+sudo data-platform-admin add-service-config my-cache-proxy cache-proxy  HOST="0.0.0.0" CACHE_PROXY_PORT="»CACHE_PROXY_PORT«" CACHE_PROXY_STATS_PORT="»CACHE_PROXY_STATS_PORT«" CACHE_TTL="15s" RIAK_KV_SERVERS="»RIAK_IP_1«:»RIAK_PB_PORT«,»RIAK_IP_2«:»RIAK_PB_PORT«" REDIS_SERVERS="»REDIS_IP_1«:»REDIS_PORT«,»REDIS_IP_2«:»REDIS_PORT«"
 ```
 The IP addresses you provide should be the IP addresses of the 3 BDP nodes you started and joined earlier.
 
 
 ##Configuration Defaults
 
-
 Each service has one or more default ports if a port has not been specified when adding a service configuration.
 
 For example, you can add a service configuration from the command line for Redis using a specified port:
 
-```
-data-platform-admin add-service-config my-redis redis HOST="10.0.0.1" REDIS_PORT="9999"
+```bash
+sudo data-platform-admin add-service-config my-redis redis HOST="0.0.0.0" REDIS_PORT="»REDIS_PORT«"
 ```
 
 Or you can use the default port by leaving out the `REDIS_PORT` parameter:
 
 ```
```

Comment:
That is correct.

Using ```` ```shell```` will sometimes work, but if we want consistent and correct syntax highlighting, we gotta use ```` ```bash````.
Or modify our render code. Which we could totally do. Just not as a part of this PR.